tests/csv-test-config.yml

```diff
@@ -23,7 +23,7 @@ fields:
     isOneOf: DEM,REP,IND,,CON,NP,LIB,NA,WIG,NA,GRE,WGR,AME
   candidate:
     - matches:
-      # Line break, letter, number ()&,".+ or spaace
-      - ^[\r\na-z0-9-()\&,"\.\+' ]+$
+      # Letter, number ()&,".+ or spaace
+      - ^[áa-z0-9-()\&,"\.\+' ]+$
```

Comment:
For Juan B. Colás, as he is known in the 2015-04-07 election. (Exactly 6 years prior, he was listed as "Juan B. Colas")